src/app/components/POCs/ProcessorTemplateSelector/ProcessorTemplateSelector.css

```diff
@@ -0,0 +1,13 @@
+.processor-template-selector,
+.processor-template-selector__preview-section {
+  height: 100%;
+}
+
+.processor-template-selector__template-selection {
+  max-width: 22em;
+}
+
+.processor-template-selector__left-section {
+  max-height: 100vh;
```

Comment:
I set it to have an internal scrollbar for the section containing the templates (in case the number of the templates will be high or when the height for containing them is not enough.
Let me try to use the `Grid` approach you suggested in https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/291#discussion_r1067361057 and let me see if I can avoid it.